src/Service/ActivityPubManager.php

```diff
@@ -108,8 +108,12 @@ public function createCcFromBody(string $body): array
      *
      * @return User|Magazine|null or Magazine or null on error
      */
-    public function findActorOrCreate(string $actorUrlOrHandle): null|User|Magazine
+    public function findActorOrCreate(?string $actorUrlOrHandle): null|User|Magazine
```

Comment:
slightly curious when this and `handle` were being called with null args, maybe that is more wrong than this accepting null?